src/Support/CacheStore.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Spatie\Ray\Support;
+
+use Carbon\CarbonImmutable;
+
+class CacheStore
+{
+    /** @var array */
+    protected $store = [];
+
+    public function hit(): self
+    {
+        $this->store[] = CarbonImmutable::now()->toAtomString();
```

Comment:
We can't use carbon as it isn't required by the package, and we don't want to add any dependencies. 

Best solution would to make a small wrapper of our own around `DateTimeImmutable`